src/util/saveArticles.ts

```diff
@@ -526,6 +534,42 @@ export function applyOtherTreatments(parsoidDoc: DominoElement, dump: Dump) {
       }
     }
   }
+}
+
+const clearNodes = (parsoidDoc: DominoElement, filtersConfig: any) => {
```

Comment:
Should we not have this in renders as part of the treatments?

Reply:
Done.